### DISCUSSION TITLE
Typo in workflow syntax for github actions documentation

### DISCUSSION THREAD

**Discussion by csumpter** [community/community#24851] [Apps, API and Webhooks]
<p>This is a report for github staff / team members. If there is a more appropriate place to submit documentation feedback, please let me know.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/fdfbb5a8e8a8a87bbe5969ebbe217f365b6129cc.png" data-download-href="/uploads/short-url/AeQ5iaK9qjZYBhXas6bD2XB4rU0.png?dl=1" title="Screen Shot 2019-09-06 at 1.36.46 PM.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/6403663/181087069-3e675e60-c9a7-4c2e-be70-f045054a4698.png" alt="Screen Shot 2019-09-06 at 1.36.46 PM.png" data-base62-sha1="AeQ5iaK9qjZYBhXas6bD2XB4rU0" width="690" height="319" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/fdfbb5a8e8a8a87bbe5969ebbe217f365b6129cc_2_690x319.png, https://aws1.discourse-cdn.com/github/original/1X/fdfbb5a8e8a8a87bbe5969ebbe217f365b6129cc.png 1.5x, https://aws1.discourse-cdn.com/github/original/1X/fdfbb5a8e8a8a87bbe5969ebbe217f365b6129cc.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/fdfbb5a8e8a8a87bbe5969ebbe217f365b6129cc_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2019-09-06 at 1.36.46 PM.png</span><span class="informations">744×345 30.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Link: <a href="https://help.github.com/en/articles/workflow-syntax-for-github-actions" rel="nofollow noopener">https://help.github.com/en/articles/workflow-syntax-for-github-actions</a></p>
<p>My interpretation is that this should be <code>github.event_name</code> <strong>NOT</strong> <code>gihub.event_name</code>.</p>

**Comment by rachmari** [comment, marked as answer]
<p><a class="mention" href="/csumpter">@csumpter</a> That is definitely a typo! I just fixed that and the latest update is live on the <a href="https://help.github.com/en/articles/workflow-syntax-for-github-actions#example-5" rel="nofollow noopener">site</a>.</p>
<p>Thanks for letting us know about this and always feel free to contact the Support team at <a href="https://support.github.com/contact" rel="nofollow noopener">https://support.github.com/contact</a> if you have suggestions or find issues in the docs.</p>
